Workbooks/TSG/categoryResources.json

```diff
@@ -0,0 +1,3 @@
+{
+    "en-us": {"name":"Troubleshooting Guides", "description": "", "order": 100}
```

Comment:
Common Troubleshooting Guided

Reply:
Should it be 'Common Troubleshooting Guide'?